packages/manager/modules/pci/src/projects/project/storages/databases/add/add.controller.js

```diff
@@ -189,7 +189,10 @@ export default class {
   }
 
   updateRegion(region) {
-    this.model.region = region;
+    if (this.model.region !== region) {
```

Comment:
`if (this.model.region.name !== region.name) {`

Reply:
Fixed